profiles/mojoshader_profile_glsl.c

```diff
@@ -2237,7 +2262,7 @@ void emit_GLSL_DSY(Context *ctx)
     output_line(ctx, "%s", code);
 } // emit_GLSL_DSY
 
-void emit_GLSL_TEXLDD(Context *ctx)
+void emit_GLSL_TEXLDD_setup(Context *ctx)
```

Comment:
This is going to sound nuts, but let's put this function at the top of `glsl_texld` inside a block, `if (texldd || texldl)`. Just trying to keep the big monolith of death in one spot to be split up reasonably later.

Reply:
Is my resolution of this OK? I moved the function and its call site. Though I need to put an if there... I'll try to do that today, I forgot.

Reply:
While putting the `if` in there, go ahead and inline it where the setup call is. Aside from that, everything else now lgtm.